### ISSUE TITLE
Error when lazy loading is enabled

### ISSUE THREAD

**Issue by Heziode** [nuxt-modules/i18n#858]
### Version
nuxt-i18n: 6.13.10
nuxt: 2.11.0

#### [mode](https://nuxtjs.org/api/configuration-mode): 
 - [ ] universal
 - [X] spa

### Nuxt-i18n configuration

My i18n config:
```
i18n: {
  locales: [
      { code: "cs", file: "cs.json", iso: "cs-CZ" },
      { code: "da", file: "da.json", iso: "da-DK" },
      { code: "de", file: "de.json", iso: "de-DE" },
      { code: "en", file: "en.json", iso: "en-US" },
      { code: "es", file: "es.json", iso: "es-ES" },
      { code: "fr", file: "fr.json", iso: "fr-FR" },
      { code: "hu", file: "hu.json", iso: "hu-HU" },
      { code: "it", file: "it.json", iso: "it-IT" },
      { code: "nl", file: "nl.json", iso: "nl-NL" },
      { code: "pl", file: "pl.json", iso: "pl-PL" },
      { code: "pt", file: "pt.json", iso: "pt-PT" },
      { code: "ru", file: "ru.json", iso: "ru-RU" },
      { code: "sk", file: "sk.json", iso: "sk-SK" },
      { code: "sv", file: "sv.json", iso: "sv-SE" },
      { code: "tr", file: "tr.json", iso: "tr-TR" },
    ],
    defaultLocale: 'en',
    lazy: true,
    langDir: "lang",
    vueI18n: "@/plugins/vue-i18n.js",
    vuex: {
      moduleName: 'i18n',
      syncLocale: true,
      syncMessages: true,
      syncRouteParams: true
    },
    detectBrowserLanguage: {
      useCookie: true,
      cookieKey: 'locale',
      alwaysRedirect: true,
      fallbackLocale: null
    },
}
```

### Reproduction Link

Check [my repo](https://github.com/Heziode/FOE-Tools.github.io/tree/feature/migrate-to-vue-i18n) (commit 2a6b6087e85940a8532db6cf94660209ae993108)

### Steps to reproduce

```
git clone https://github.com/Heziode/FOE-Tools.github.io lazy-load-error

cd lazy-load-error

yarn install

yarn dev

# Navigate to http://localhost:3000
```

### What is Expected?

The main page should be displayed, without errors.

### What is actually happening?

I get this error:

```
Cannot read property '__i18n' of undefined
```

**Comment by Heziode** [comment]
The problem seems to be fixed by upgrading to **nuxt 2.14.3**.
Nowhere is it mentioned that it requires a certain version of nuxt.

**Comment by rchl** [comment]
Looks like a bug. Not sure why the Nuxt version makes a difference. Probably something subtle has changed.

**Comment by rchl** [comment]
Released fix in [v6.13.11](https://github.com/nuxt-community/i18n-module/releases/tag/v6.13.11)
